core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/Benchmark.scala

```diff
@@ -84,54 +73,25 @@ class Benchmark(
    * a comment with the benchmark. Although the results vary from machine to machine, it should
    * provide some baseline.
    */
-  def run(): Unit = {
+  def run(): mutable.ArrayBuffer[Result] = {
```

Comment:
Can we return `Seq[Result]`? As the caller does not modify the results, we do not need `mutable.ArrayBuffer`